src/test/kotlin/chapter8/exercises/ex14/listing.kt

```diff
@@ -4,12 +4,20 @@ import chapter8.sec3.listing3.Gen
 import chapter8.sec3.listing3.Prop
 import chapter8.sec4.listing1.run
 
-val smallInt = Gen.choose(-10, 10)
+class Helper {
+    //tag::init[]
```

Comment:
The book might need to be updated to include only `[tag=init]`

Reply:
Not sure what you mean. `[tag=init]` isn't a valid asciidoc tag.

Reply:
Not sure what you mean. `[tag=init]` isn't a valid asciidoc tag.

Reply:
@marc0der, sorry for confusion. I know next to nothing about asciidoc, I just assumed it works this way:

1. Wrap a snippet of code with two lines: `//tag::nameOfYourTag[]` `//end::nameOfYourTag[]`
2. Include code snipped in asciidoc with `include::listing.kt[tag=nameOfYourTag]`

https://asciidoctor-docs.netlify.com/asciidoc/1.5/directives/include-lines-and-tags/

I don't know if `init` has any special meaning, I just saw it in many places in the code. So, when I wrote `[tag=init]`, I meant `include::listing.kt[tag=init]`.